packages/auth/src/authClients/magicLink.ts

```diff
@@ -1,10 +1,8 @@
+import type { AuthClient } from './'
 import type { Magic, MagicUserMetadata } from 'magic-sdk'
```

Comment:
This is Prettier's formatting, which ESLint doesn't like:
<img width="554" alt="Screen Shot 2021-05-14 at 12 22 16 PM" src="https://user-images.githubusercontent.com/2951/118319017-06b90f80-b4af-11eb-89b4-56b359ca0554.png">